### PR TITLE
Footer v1.1

### PR DESCRIPTION
**Pull request by vrajparesh**
Resized al footerl elements according to the given design

### CODE REVIEW

**Comment by rossoskull** [comment]
Please deploy it to [surge](https://surge.sh/) and comment the link. Contact @pAditya198 for help

**Comment by vrajparesh** [comment]
@rossoskull please review this.

link:  http://cerebro-footer.surge.sh/

<blockquote><div><strong><a href="http://cerebro-footer.surge.sh/">Cerebro</a></strong></div><div>Naya cerebro</div></blockquote>

File: package.json [hunk 1]
```diff
@@ -6,18 +6,21 @@
...
     "@testing-library/user-event": "^7.1.2",
+    "core-js": "^3.6.4",
     "node-sass": "^4.13.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.3.0"
+    "react-scripts": "3.3.0",
+    "surge": "^0.21.3"
```
Comment (rossoskull) [inline package.json]: Remove surge from local package, add it globally to your system by `npm i -g surge`

File: src/Components/Footer/Footer.jsx [hunk 2]
```diff
@@ -3,38 +3,64 @@ import './Footer.scss';
...
-                        <div className="containerTitle" ><h2>Contact</h2><div className="hr-text"><hr color="#29abe2" /></div><br /><br /></div>
-                    </div>
-                    <div className="links_div"><a class="Linkname" >Email: cerebro@iiitv.ac.in</a></div>
-                    <div className="links_div"><a class="Linkname" >Phone: +91-9370645038</a></div>
-                    <div className="links_div"><a class="Linkname" >Address: c/o Block No.9, Government Engineering<div className="links_div"> College, Sector-28, Gandhinagar, Gujarat - 382028</div></a></div>
-                </div>
-            </div>
-        </div>
-    );
+		// 		<div className="logo">
```
Comment (rossoskull) [inline src/Components/Footer/Footer.jsx]: Remove comments

File: src/Components/Footer/Footer.scss [hunk 3]
```diff
@@ -1,103 +1,228 @@
...
-    padding-left: 3rem;
-    //grid-auto-rows:minmax(auto, auto);
-    grid-auto-columns: min-content;
-}
+.footer {
+    display: flex;
+    flex-flow: row;
+    justify-content: space-between;
+    border-top: 2px solid grey;
+    width: 1808px;
```
Comment (rossoskull) [inline src/Components/Footer/Footer.scss]: Don't use a fixed width

File: src/Components/Footer/Footer.scss [hunk 4]
```diff
@@ -1,103 +1,228 @@
...
-.hr-text {
-    width: 3rem;
-    padding-top: 1rem;
-    padding-left: 0.8rem;
-    height: 0rem;
-    
+// .footer-container {
 
-}
+//     padding: 48px 80px 16px 16px;
```
Comment (rossoskull) [inline src/Components/Footer/Footer.scss]: Comments

**Review by rossoskull — CHANGES_REQUESTED** [review]

**Comment by vrajparesh** [comment]
WhatsApp Image 2020-02-02 at 17 41 29
Portrait view
WhatsApp Image 2020-02-02 at 17 41 29 (1)
Landscape(Rotated) view

**Comment by pAditya198** [comment]
There are 6 commits so squash them into 1

**Comment by pAditya198** [comment]
@vrajparesh

**Review by rossoskull — APPROVED** [review]

**Pull request merged**